src/amber/controller/error.cr

```diff
@@ -12,13 +12,7 @@ module Amber::Controller
     end
 
     def internal_server_error
-      response_format("ERROR: #{@ex.inspect_with_backtrace}")
```

Comment:
I think we should leave these comments here or else someone will refactor it again because they won't understand why there's added complexity here.